workers/unity/Assets/Gdk/Core/Worker/WorkerBase.cs

```diff
@@ -19,18 +19,22 @@ public abstract class WorkerBase : IDisposable
 
         public abstract string GetWorkerType { get; }
 
-        protected WorkerBase(string workerId, Vector3 origin)
+        protected WorkerBase(string workerId, Vector3 origin) : this(workerId, origin, new LoggingDispatcher())
+        {
+        }
+
+        protected WorkerBase(string workerId, Vector3 origin, ILogDispatcher loggingDispatcher)
         {
             if (string.IsNullOrEmpty(workerId))
             {
-                throw new ArgumentException("WorkerId is null or empty.");
+                throw new ArgumentException("WorkerId is null or empty.", workerId);
```

Comment:
The second parameter is meant to be the name of the parameter that was empty, so you want `nameof(workerid)`